i18n/locales/source/course_content/2018/coursee-2018.json

```diff
@@ -1463,7 +1463,21 @@
   },
   "https://studio.code.org/s/coursee-2018/stage/21/puzzle/2": {
     "short_instructions": "Change the behavior inside the `___ begins ___` block to make the alien start spinning around.",
-    "long_instructions": "The alien was proud of its dancing skills on Mars, but now it can only move side-to-side.  \n\nCan you change the behavior inside the [blockly block][0] block to make it start spinning around?  "
+    "long_instructions": "The alien was proud of its dancing skills on Mars, but now it can only move side-to-side.  \n\nCan you change the behavior inside the [blockly block][0] block to make it start spinning around?  ",
+    "behavior_names": {
+      "shrinking": "shrinking",
+      "patrolling": "patrolling",
+      "wandering": "wandering",
+      "growing": "growing",
+      "jittering": "jittering",
+      "moving east": "moving east",
+      "moving north": "moving north",
+      "moving south": "moving south",
+      "moving west": "moving west",
+      "spinning right": "spinning right",
+      "spinning left": "spinning left",
+      "swimming left and right": "swimming left and right"
+    }
```

Comment:
@bethanyaconnor we should probably pull these out into a standalone file, rather than having them alongside the other course content

Reply:
Why? We put function names in the course content.

Reply:
the primary difference being that function names are generally unique to a level, whereas these looks more like category names in that there seems to be a lot of repeats between levels

Reply:
I'd rather think about how to clean up the levels. Some levels do have behaviors specific to the level (e.g. `being hungry` or `mystery behavior`) and it seems like most of the levels with these lists of behaviors don't actually use them.

Reply:
oh yeah, that sounds much better!